### ISSUE TITLE
[enhancement request] set camera_control_button_resistance

### ISSUE THREAD

**Issue by nerdCopter** [emuflight/EmuFlight#440]
EmuFlight does not have `camera_control_button_resistance`

**Comment by Bigguy34** [comment]
Hey @nerdCopter can I work on this? I also am working on testing fast sbus.

**Comment by nerdCopter** [comment]
We welcome Pull Requests if you code :)

**Comment by Quick-Flash** [comment]
> Hey @nerdCopter can I work on this? I also am working on testing fast sbus.

yes please, just try and find where bf made a pull request for this and you will probably be able to mostly just copy it over.

**Comment by Bigguy34** [comment]
Yep @Quick-Flash that is what I was planning on doing thanks!

**Comment by nerdCopter** [comment]
presumably fixed via #1171
